scripts/run_pipeline.py

```diff
@@ -33,32 +35,44 @@ def _run_cmd(cmd, cwd, input_file, output_file):
 
 def _extract_module(input_module, input_file_mode, output_base_dir, cwd):
     # Tactic prediction
-    _run_cmd(
-        cmd='training_data',
-        cwd=cwd,
-        input_file=input_module,
-        output_file=os.path.join(
-            output_base_dir, 
-            DIR_NAMES['training_data'],
-            _get_stem(input_module, input_file_mode) + '.jsonl'
-        )
-    )
+    # _run_cmd(
```

Comment:
Perhaps we can add a command line flag to determine which of these is run (rather than commenting out)